### PR TITLE
Adds Round 3 Synthesis 

### PR DESCRIPTION
**Pull request by reitermb**
### This pull request changes...

Adds Round 3 Synthesis

### CODE REVIEW

**Comment by alexsoble** [comment]
Dropping in this preview link real quick for ease of review: https://github.com/reitermb/TANF-app/blob/main/docs/User-Research/2020%2C%20Fall%20-%20Understanding%20STT%20Roles%2C%20Source%20of%20Truth%2C%20and%20Metadata.md

File: docs/User-Research/2020, Fall - Understanding STT Roles, Source of Truth, and Metadata.md [hunk 1]
```diff
@@ -0,0 +1,450 @@
...
+
+# Who we talked to
+
+Given the largely generative focus of this research we wanted to talk to 5-9 participants involved in TANF data preparation from a variety of grantee types. We targeted criteria such as: 
+
+- At least one state that submits universal case data
+- At least one state that submits sample data
+- At least one tribe
+- At least one territory
+- Preference toward participants who have no been interviewed in past research
```
Comment (lfrohlich) [inline docs/User-Research/2020, Fall - Understanding STT Roles, Source of Truth, and Metadata.md:70]: who have noT been

File: docs/User-Research/2020, Fall - Understanding STT Roles, Source of Truth, and Metadata.md [hunk 2]
```diff
@@ -0,0 +1,450 @@
...
+- "...I would do four different transmissions, you know, put this one up, send it and then do the active, and then closed and then the aggregate"
+- re. upload/. conceptual prototype : "Cause it might just be two [quarters] that we're doing now"
+
+Some grantees submit monthly reports instead of quarterly 
+
+- "... we've been trying to do now is do [TANF report] each month separately as we finish it, instead of doing the whole quarter at once"
+
+**Project Impacts**
+- Consider how we can make the upload function flexible and generally accommodate differing flows relating to assembling TANF reports.
+- Include notifications to alert a user of an incomplete report if the upload sections at different times.
```
Comment (lfrohlich) [inline docs/User-Research/2020, Fall - Understanding STT Roles, Source of Truth, and Metadata.md:291]: ...if theY upload sections...

File: docs/User-Research/2020, Fall - Understanding STT Roles, Source of Truth, and Metadata.md [hunk 3]
```diff
@@ -0,0 +1,450 @@
...
+
+
+
+
+
+### Grantees understand the term 'encryption' differently. 
+
+Some grantees seemed to understood encryption as being something roughly equivalent to 'non human-readable'. Others did connect our question about encryption with the TANF header data point regarding social security numbers (SSN). However not all grantees encrypt those, instead relying on security inherent to the SFTP transfer processes (this includes managed SFTP vendor services like Cyberfusion):
```
Comment (lfrohlich) [inline docs/User-Research/2020, Fall - Understanding STT Roles, Source of Truth, and Metadata.md:299]: some grantees seemed to underSTAND...

**Review by lfrohlich — CHANGES_REQUESTED** [review]
Noted areas for minor edits

**Comment by reitermb** [comment]
Thanks @lfrohlich—Should be updated now!

**Review by lfrohlich — APPROVED** [review]
Thanks!

**Pull request merged**